includes/class-wc-payments-features.php

```diff
@@ -254,6 +254,7 @@ public static function to_array() {
 				'woopayExpressCheckout'   => self::is_woopay_express_checkout_enabled(),
 				'isAuthAndCaptureEnabled' => self::is_auth_and_capture_enabled(),
 				'progressiveOnboarding'   => self::is_progressive_onboarding_enabled(),
+				'isNewDepositsOverview'   => false,
```

Comment:
Nah, I had a note in the PR description but it could've been missed.

The real feature flag is being implemented by https://github.com/Automattic/woocommerce-payments/issues/5690